### ISSUE TITLE
[Enhancement] Minimal toolset dependency MAUI application

### ISSUE THREAD

**Issue by mrange** [dotnet/maui#1392]
## Summary
I like the capability to build a MAUI application target at desktop (Windows, Mac or Linux) with minimal toolset dependency.

The background for this is that I thought I wanted to try MAUI for building a small utility application for my Windows, Mac and Linux boxes. I don't care about mobile device in my use case.

When looking at: https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation I got discouraged due the sheer size of toolset dependencies I need to add.

AFAICT I need:

- Mobile development with .NET
- Universal Windows Platform development
- Desktop development with C++
- .NET Desktop Development
- Xamarin
- Project Reunion (Preview)
- Single-project MSIX Packaging Tools
- OpenJDK 11
- OpenJDK 8
- Android SDK
- Various Microsoft SDKs for mobile devices.

I am a difficult person and I don't want to install all these dependencies and those my MAUI experiment stopped there. 

What I envisioned which might not be a relevant vision in 2021 is that I could install a MAUI runtime with cherry-picked dependencies for Desktop development. I could then develop a MAUI app to target desktop devices which is the only thing relevant in my use case.

I realize I am just one guy and my use case might be considered irrelevant so feel free to close the ticket. However, it could be that others share a similar sentiment so I wanted at least to create a ticket to share my opinion on the matter.

I am very light on the actual changes needed to be done as I don't understand MAUI well enough to give any pointers.

Fundamentally I want to get started with MAUI for desktop apps faster than I can get started with Windows Forms development.

## API Changes

Not sure.

## Intended Use Case

Described in the summary

**Comment by djsell** [comment]
Yes. Having to install OpenJDK + Android SDK in order to not develop any Java or Android apps is not ideal.

**Comment by manuelduarte077** [comment]
Well, I agree with you, many things have to be installed to do something, I have been testing MAUI for a few days, but installing many things takes away my desire to continue, when other platforms are more careful with that.
